### ISSUE TITLE
Windows search paths are case-sensitive

### ISSUE THREAD

**Issue by moswald** [Microsoft/vscode#48953]
Issue Type: <b>Bug</b>

I did a search on my full repository, and got several hits under `A\B\C`. I set my path filter to `a\b` and got nothing. Only after switching to `A\B` did any results come back.

This is probably expected behavior on OS X/Linux (although I would perhaps argue that it's not user-friendly even on those platforms), but on Windows paths are generally expected to be case-insensitive.

VS Code version: Code 1.22.2 (3aeede733d9a3098f7b4bdc1f66b63b0f48c1ef9, 2018-04-12T16:38:45.278Z)
OS version: Windows_NT x64 10.0.17127

<details><summary>Extensions (26)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|0.19.1
npm-intellisense|chr|1.3.0
vscode-markdownlint|Dav|0.15.0
githistory|don|0.4.0
xml|Dot|1.9.2
gitlens|eam|8.2.4
vscode-html-css|ecm|0.2.0
EditorConfig|Edi|0.12.1
tslint|eg2|1.0.28
vscode-npm-script|eg2|0.3.4
auto-close-tag|for|0.5.6
beautify|Hoo|1.3.0
rest-client|hum|0.18.3
mssql|ms-|1.3.1
azure-account|ms-|0.4.0
cpptools|ms-|0.16.1
csharp|ms-|1.14.0
PowerShell|ms-|1.7.0
debugger-for-chrome|msj|4.3.0
vscode-docker|Pet|0.0.26
material-icon-theme|PKi|3.3.0
java|red|0.23.0
vscode-icons|rob|7.23.0
preview-vscode|sea|1.4.2
code-settings-sync|Sha|2.9.0
html-css-class-completion|Zig|1.17.1

</details>

**Comment by roblourens** [comment]
/duplicate of https://github.com/Microsoft/vscode/issues/10633
